package.json

```diff
@@ -26,7 +26,7 @@
     "check-changed-files": "yarn ts-node scripts/ts-check-changed-files.ts"
   },
   "engines": {
-    "node": "14.x",
+    "node": ">=14.x",
```

Comment:
There may be a problem using 16 with Railway deployments. I don't remember exactly, maybe @emrysal knows.

Reply:
There is, but on the other hand we should be looking at the Railway deployment itself; why is it failing; in addition there is many other reasons it fails at the moment due to outdated env vars, needs updating.